### DISCUSSION TITLE
AP2: could it possibly work on OSMC?

### DISCUSSION THREAD

**Discussion by TheManchineel** [mikebrady/shairport-sync#1239] [General]
Is there any way AP2 could be made function on OSMC for Raspberry Pi? Having AirPlay 2 + Kodi + Plex support on a single Raspberry Pi 3 would be amazing

**Comment by charlesomer** [comment]
Are you referring to the OSMC App Store? Kodi currently supports AirPlay 1 as far as I'm aware so I guess it's possible that could be updated to AirPlay 2 at some point in the future but I'm not sure if that will happen

**Comment by iChUdY** [comment]
Just install development shairport-sync over ssh. But must configure Kodi to as soon possible out from sound card after stop Kodi media. Kodi and shairport-sync will argue about the sound card.
